src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Models/ScheduleDisplay.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Microsoft.Azure.Management.DevTestLabs.Models;
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.Commands.DevTestLabs.Models
+{
+    internal class ScheduleDisplay : Schedule
+    {
+        internal static string ToLocalizedTime(string militaryTime)
+        {
+            if (militaryTime == null || militaryTime.Length != 4)
+            {
+                return militaryTime;
+            }
+
+            ushort hours;
+
+            if (!ushort.TryParse(militaryTime.Substring(0, 2), out hours))
+            {
+                return militaryTime;
+            }
+
+            ushort minutes;
+
+            if (!ushort.TryParse(militaryTime.Substring(2), out minutes))
+            {
+                return militaryTime;
+            }
+
+            if (hours > 23 || minutes > 59)
+            {
+                return militaryTime;
+            }
+
+            return new DateTime(2000, 1, 1, hours, minutes, 0, 0).ToShortTimeString();
+        }
+
+        public new WeekDetailsDisplay WeeklyRecurrence
```

Comment:
Output types should use proeprties with easily usable types in PowerShell, e.g., string, int, array, DateTime